server/sendChatMessage/index.ts

```diff
@@ -62,6 +115,52 @@ const httpTrigger: AzureFunction = async function (
       return
     }
 
+    const danceMatch = /^\/(dance)(.*)/.exec(message)
+    if (danceMatch) {
+      context.bindings.signalRMessages = [
+        {
+          groupName: user.roomId,
+          target: 'dance',
+          arguments: [req.body.id, user.id, danceList[Math.floor(Math.random() * danceList.length)]]
+        }
+      ]
+      return
+    }
+
+    const interactMatch = /^\/(interact|get) (.*)/.exec(message)
+    // Todo, would be great if this could live in its own file, since it's just gonna get bigger
+    // Todo, limit these to the right room
+    if (interactMatch) {
+      var inspectedObject = interactMatch[2]
+      if (inspectedObject.includes('cookie') || inspectedObject.includes('fortune')) {
+        // Inspecting a fortune cookie
+        context.bindings.signalRMessages = [
+          {
+            groupName: user.roomId,
+            target: 'emote',
+            arguments: [req.body.id, user.id, ('opens a fortune cookie that reads: ' + fortuneList[Math.floor(Math.random() * fortuneList.length)])]

```

Comment:
@lazerwalker another question, if you have the time. Kawa had included some fortune cookies with links in them to previous talks, the code of conduct, etc. They wrote on the text for the fortune as HTML with the hyperlink - but the way this gets rendered in the end it escapes the characters and instead of a hyperlink you can text with the unicode mess and the href etc. Looking into options if I did want to try to render the text as HTML it all looks pretty messy, like using "dangerouslySetInnerHTML" - do you know of any way you'd be happy with? Low priority, since we don't *need* links in fortunes.

Reply:
We're currently using dangerouslySetInnerHTML for rendering the room descriptions (plus some manual parsing for the Twine-style links), I think it's fine if we use it for this case. 

It's mostly a vulnerability if you're using it to render user-generated text, but as long as we're only using it for content that's hardcoded into our server code it should be safe.

Reply:
Okay awesome - I'll try that out tonight!